zcash_client_backend/src/data_api.rs

```diff
@@ -1369,6 +1365,8 @@ pub trait WalletRead {
     fn get_transparent_receivers(
         &self,
         _account: Self::AccountId,
+        _include_change: bool,
```

Comment:
Do we create, or can we reasonably anticipate creating, internal *transparent* addresses other than for change? I think auto-shielding and note management will not do so.

Reply:
We will generate these as part of gap limit handling in order to make it possible to recover the full history for transparent-only wallets that follow the Bitcoin conventions for generation of change addresses.

Reply:
I think the intent of the question was missed. It was: is it always correct to refer to these internal addresses as "change addresses", or are there / will there be non-change internal addresses (as there are in the shielded case)?

Reply:
By definition these are addresses at the `change` child index.